os-certmonger.te

```diff
@@ -11,4 +11,4 @@ read_files_pattern(certmonger_t, puppet_etc_t, puppet_etc_t)
 
 # rhbz#1777368
 container_runtime_domtrans(certmonger_t)
-container_runtime_entrypoint(certmonger_t)
+allow certmonger_t container_runtime_exec_t:file entrypoint;
```

Comment:
hmmm we might get issues if we don't import the "container_runtime_exec_t" type.. ? @jpichon ?
But I was about pushing something like that :).

Might even better to actually get an up-to-date container-selinux version though... ?